src/core/Microsoft.PowerToys.Settings.UI/Views/ImageResizerPage.xaml

```diff
@@ -40,86 +45,136 @@
             <RowDefinition Height="Auto"/>
             <RowDefinition Height="Auto" />
         </Grid.RowDefinitions>
-        <StackPanel Orientation="Vertical">
-            <TextBlock Text="Lets you resize images by right-clicking."
+        <StackPanel Orientation="Vertical" x:Name="ImageResizerView">
+
+            <TextBlock x:Uid="ImageResizer_Description"
                        TextWrapping="Wrap"/>
 
-            <ToggleSwitch Header="Enable Image Resizer"
-                          IsOn="True"
+            <TextBlock x:Uid="ImageResizer_EnableGroupSettingsHeader"
+                       Style="{StaticResource SettingsGroupTitleStyle}"/>
+
+            <ToggleSwitch x:Uid="ImageResizer_EnableToggle"
+                          IsOn="{ Binding Mode=TwoWay, Path=IsEnabled}"
                           Margin="{StaticResource SmallTopMargin}" />
 
-            <TextBlock Text="Sizes" 
+            <TextBlock x:Uid="ImageResizer_CustomSizes"
                        Style="{StaticResource SettingsGroupTitleStyle}"/>
 
-            <!-- TO DO: Don't know if a listview with these items is the best solution here. Maybe a data grid is better? -->
-            <!--<ListView Margin="{StaticResource SmallTopMargin}" Padding="-12,0,0,0" SelectionMode="None" ItemsSource="{x:Bind Sizes, Mode=OneWay}">
+
+            <ListView x:Name="ImagesSizesListView" 
+                      ItemsSource="{Binding Sizes, Mode=TwoWay, Source={StaticResource ViewModel}}"
+                      IsEnabled="{ Binding Mode=TwoWay, Path=IsEnabled}">
                 <ListView.ItemTemplate>
-                    <DataTemplate x:DataType="local:ResizeSize">
-                        <StackPanel Orientation="Horizontal" Spacing="4">
-                            <TextBox Text="{x:Bind Title}" Width="68"/>
-                            <ComboBox SelectedIndex="1" Width="88">
+                    <DataTemplate x:Name="SingleLineDataTemplate" x:DataType="models:ImageSize">
+                        <StackPanel Orientation="Horizontal" Spacing="4" >
+                            <TextBox Text="{x:Bind Name}" 
+                                     Width="90" 
+                                     Height="35" />
+
+                            <ComboBox 
+                                      SelectedIndex="{x:Bind Fit}"
+                                      Width="90" 
+                                      Height="35">
                                 <ComboBoxItem>Fill</ComboBoxItem>
                                 <ComboBoxItem>Fit</ComboBoxItem>
                                 <ComboBoxItem>Stretch</ComboBoxItem>
```

Comment:
This aren't internationalized. In the PowerLauncher page there is an example how you can do that with the `ComboBox`